.travis.yml

```diff
@@ -1,7 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - "6.0"
```

Comment:
I dropped 6 because it fails on Travis now due to some internal dependency versions. This problem is avoided in 8 and up because of package-lock.json.